src/main/resources/templates/project-view.ftl

```diff
@@ -1,135 +1,143 @@
-								<span class="state glyphicon glyphicon-ok-circle" title="Build succeeded!"></span>
-							</a>
-					[#else]
-						<td class="commit failed" id="${commit.getCommit()}">
-							<a href="/projects/${group.course.code}/groups/${group.groupNumber}/commits/${commit.getCommit()}/build">
-								<span class="state glyphicon glyphicon-remove-circle" title="Build failed!"></span>
-							</a>
-					[/#if]
-				[#else]
-						<td class="commit running" id="${commit.getCommit()}">
-							<span class="state glyphicon glyphicon-align-justify" title="Build queued..."></span>
-				[/#if]
-			[#else]
-						<td class="commit ignored" id="${commit.getCommit()}">
-							<span class="state glyphicon glyphicon-unchecked"></span>
-			[/#if]
-							<a href="/projects/${group.course.code}/groups/${group.groupNumber}/commits/${commit.getCommit()}/diff">
-								<div class="comment">${commit.getMessage()} [@listTags repository commit.getCommit() /]</div>
-								<div class="committer">${commit.getAuthor()}</div>
-								<div class="timestamp" data-value="${(commit.getTime() * 1000)?c}">on ${(commit.getTime() * 1000)?number_to_datetime?string["EEEE dd MMMM yyyy HH:mm"]}</div>
-							</a>
-						</td>
-					</tr>
-		[/#list]
-	[#else]
-						<tr>
-							<td class="muted">
-								There are no commits in this repository yet!
-							</td>
-						</tr>
-	[/#if]
-[#else]
-						<tr>
-							<td class="muted">
-								Could not connect to the Git server!
-							</td>
-						</tr>
-[/#if]
-				</tbody>
-			</table>
+            <h4 style="line-height:34px; margin-top:0;">Recent commits</h4>
+        </div>
+    </div>
 
+    <div class="row">
+        <div class="col-md-2">
+        [@projectFrameset.renderSidemenu "commits" i18n group repository/]
+        </div>
+        <div class="col-md-10">
 
-[#function max x y]
-    [#if (x<y)][#return y][#else][#return x][/#if]
-[/#function]
+            <table class="table table-bordered" id="table-commits">
+                <tbody>
+                [#if repository?? && repository?has_content]
+                    [#if commits?? && commits?has_content]
+                        [#list commits.commits as commit]
+                            [@commitRow.render group states commit.commit "/courses/${group.course.code}/groups/${group.groupNumber}/commits/${commit.commit}/diff"]
+                                [#if comments?? && comments?has_content]
+                                    [#assign numComments = comments.amountOfCommits(commit.commit)]
+                                    [#if numComments > 0]
+                                        <span class="pull-right"><i class="glyphicon glyphicon-comment"></i> ${numComments}</span>
+                                    [/#if]
+                                [/#if]
+                                <div class="comment">${commit.getMessage()} [@listTags repository commit.getCommit() /]</div>
```

Comment:
Let's make a separate feature request (issue) of this. I see the benefit but I probably won't have the time to fix this for the next deployment.